libpod/sql_state_internal.go

```diff
@@ -170,6 +170,7 @@ func prepareDB(db *sql.DB) (err error) {
             MountLabel TEXT NOT NULL,
             Mounts TEXT NOT NULL,
             ShmDir TEXT NOT NULL,
+            ShmSize TEXT NOT NULL,
```

Comment:
Is TEXT right here and string right below?  Or should they be INTEGER and int?

Reply:
THey are passed around and used as a string.  You can use indicators in the string like
52M or 65k